app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A simple, Unicode-aware word processor that runs on the console"
+DESCRIPTION="WordGrinder is designed to get out of your way and let you write; \
+it does very little, but what it does it does it well. It supports basic paragraph \
+styles, basic character styles, basic screen markup, a menu interface that means you \
+don't have to remember complex key sequences, HTML import and export, and some other \
+useful features."
+HOMEPAGE="http://cowlark.com/wordgrinder/"
+COPYRIGHT="2007-2017 David Given"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/davidgiven/wordgrinder/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="54085af7963e1f67342bc0b1b20d1ccc75494f2e23d401b601cf3089acea747c"
+SOURCE_DIR="wordgrinder-$gitSrcRevision"
```

Comment:
```diff
-SOURCE_DIR="wordgrinder-$gitSrcRevision"
```
(We only define SOURCE_DIR if its default value is not correct.)
I can't run Haiku at the moment, so I won't be able to test.

Regarding ncurses, maybe you could try ncurses6 (instead of ncurses).
Regarding Lua, you might wish to try compiling against Lua 5.3.

Reply:
Whoops. I meant to change that :0
For lua, would I use liblua5?